.github/workflows/dev-workflow.yml

```
@@ -0,0 +1,68 @@
+
+      - name: Deploy APIs
+        env:
+          REGION: ap-northeast-2
+          ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        run: |
+          env
+          npm i -g aws-cdk tailwindcss
+          ENV=dev make deploy
+
+  create-release-pr:
```

actionlint

job "create-release-pr" needs job "canister-build-deployment" which does not exist in this workflow

(job-needs)